cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -174,6 +175,15 @@ func VerifyBlobCmd(ctx context.Context, ko sign.KeyOpts, certRef, sigRef, blobRe
 		}
 		fmt.Fprintln(os.Stderr, "Certificate is trusted by Fulcio Root CA")
 		fmt.Fprintln(os.Stderr, "Email:", cert.EmailAddresses)
+		for _, uri := range cert.URIs {
+			fmt.Fprintf(os.Stderr, "URI: %s://%s%s\n", uri.Scheme, uri.Host, uri.Path)
+		}
+		for _, ext := range cert.Extensions {
+			oidc := asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 1}
```

Comment:
I think we might be able to reuse this: https://github.com/sigstore/cosign/blob/7066f122b8288e5621cf7e3dca600075bb70cd47/pkg/signature/keys.go#L219

Reply:
Done